### PR TITLE
DCOS-10987: [1/2] Separate JSON Flow

### PR DESCRIPTION
**Pull request by wavesoft**
This is the first PR for the separate JSON flow that cleans-up the `NewCreateServiceModalServicePicker`, keeping only the 3 acceptable options for now, and adding a placeholder for the JSON-only view.

@jfurrow I tried to center the buttons for making them look like the design, but I didn't want to mess-up the CSS. Can you have a look? https://mesosphere.invisionapp.com/d/main#/console/9042004/193660846/preview

### CODE REVIEW

**Comment by jfurrow** [comment]
@wavesoft I pushed a commit, I hope you don't mind!

It should now look like this:

**Comment by wavesoft** [comment]
Perfect, thank you!

File: plugins/services/src/js/components/modals/NewCreateServiceModal.js [hunk 1]
```diff
@@ -58,21 +66,31 @@ class NewServiceFormModal extends Component {
       return;
     }
 
+    if (serviceReviewActive) {
+      // Just hide review screen. Form or JSON mode will be
+      // activate dautomaticaly depending on their last state
```
Comment (mlunoe) [inline plugins/services/src/js/components/modals/NewCreateServiceModal.js]: #nit spelling

File: plugins/services/src/js/components/modals/NewCreateServiceModal.js [hunk 2]
```diff
@@ -58,21 +66,31 @@ class NewServiceFormModal extends Component {
...
+      this.setState({
+        serviceReviewActive: false
+      });
+      return;
```
Comment (mlunoe) [inline plugins/services/src/js/components/modals/NewCreateServiceModal.js]: #nit newline before `return`

Comment (mlunoe) [inline plugins/services/src/js/components/modals/NewCreateServiceModal.js]: We could also just set state at the bottom and let these `if`s determine the updated state

File: plugins/services/src/js/components/modals/NewCreateServiceModalServicePicker.js [hunk 3]
```diff
@@ -1,44 +1,19 @@
-import mixin from 'reactjs-mixin';
 /* eslint-disable no-unused-vars */
 import React from 'react';
 /* eslint-enable no-unused-vars */
-import {StoreMixin} from 'mesosphere-shared-reactjs';
 
-import CosmosPackagesStore from '../../../../../../src/js/stores/CosmosPackagesStore';
+// import CosmosPackagesStore from '../../../../../../src/js/stores/CosmosPackagesStore';
```
Comment (mlunoe) [inline plugins/services/src/js/components/modals/NewCreateServiceModalServicePicker.js]: Let's just remove this and re-add it when we need it

File: plugins/services/src/js/components/modals/NewCreateServiceModal.js [hunk 4]
```diff
@@ -1,11 +1,18 @@
 import React, {Component, PropTypes} from 'react';
 
+import Application from '../../structs/Application';
+import Pod from '../../structs/Pod';
+
+import {NEW_APP_DEFAULTS} from '../../constants/NewApplicationDefaults';
+import {NEW_POD_DEFAULTS} from '../../constants/NewPodDefaults';
```
Comment (nLight) [inline plugins/services/src/js/components/modals/NewCreateServiceModal.js]: ❤️ 💪

**Comment by nLight** [comment]
@wavesoft there's few nits from Michael, please consider them and merge 👍

**Pull request merged**